Sources/Private/Experimental/ExperimentalAnimationLayer.swift

```diff
@@ -38,20 +38,28 @@ final class ExperimentalAnimationLayer: CALayer {
 
   // MARK: Internal
 
-  // TODO: Add more configuration, like `loopMode`
-  func playAnimation() {
+  /// Timing-related configuration to apply to this layer's child `CAAnimation`s
+  struct TimingConfiguration {
```

Comment:
Is this effectively implementing `CAMediaTiming`? Wondering if we would want to make this explicit via a conformance or at least document it

Reply:
interesting idea -- I don't think all of the `CAMediaTiming` properties are applicable here (e.g. `duration`) but mentioning it in the doc comment can't hurt